server/test/unit/common/config/__init__.py

```diff
@@ -0,0 +1,127 @@
+import os
+import shutil
+import unittest
+import random
+from unittest import mock
+
+from server.test import FIXTURES_ROOT
+
+
+def mockenv(**envvars):
+    return mock.patch.dict(os.environ, envvars)
+
+
+class ConfigTests(unittest.TestCase):
```

Comment:
There aren't any tests in this class are there? I see helper functions to return configs but not really any assertions?

The equivalent file has the function `handle_config_from_secret`. It might be nice to test that while mocking out AWS SecretsManager?

Reply:
I have a test for `handle_config_from_secret` in `test_server_config` https://github.com/chanzuckerberg/cellxgene/pull/1854/files#diff-4cf6f8eb2af82657133e9e9cf28dedccR297
I can move it into the init file if you think that would be more clear. 

I typically think of the init file for test directories as a place to store helper functions shared across many tests. Do you prefer to store those elsewhere?

Reply:
Ah I was just confused since it inherits `unittest.TestCase`